src/pages/home/report/ReportActionCompose.js

```diff
@@ -207,9 +211,12 @@ class ReportActionCompose extends React.Component {
                         )}
                     </AttachmentModal>
                     <TouchableOpacity
-                        style={[styles.chatItemSubmitButton, styles.buttonSuccess]}
+                        style={[styles.chatItemSubmitButton,
+                            this.state.isCommentEmpty.length > 0
```

Comment:
This should just reference `this.state.isCommentEmpty`

Reply:
done